app/app.py

```diff
@@ -2,28 +2,44 @@
 import logging
 import os
 from pathlib import Path
-from subprocess import check_call
+from http import HTTPStatus
 
 from flask import Flask, request, abort
 
+from .commands import ConnectTo, NotConfigured
+
+# Add/change the deployments here!
+DEPLOYMENTS = {
+    "gunaha": NotConfigured(),
+    "hello": ConnectTo("itw.altlab.dev").command("/opt/hello.altlab.dev/deploy"),
+    "itwewina": ConnectTo("itwewina@itw.altlab.dev").command(
+        "/opt/docker-compose/itwewina/cree-intelligent-dictionary/docker/deploy"
+    ),
+    "korp-frontend-dev": ConnectTo("kor.altlab.dev").command(
+        "/etc/docker/compose/korp/deploy"
```

Comment:
Haha, `/etc/` is not the right place to put apps!

Reply:
I don't remember why it wound up there (@andrewdotn might), but I can change this after I update Korp's deployment script. Ideally the code here would pull the latest from the korp-frontend repo into `/opt/`, then build the Docker container from that (at least if I understand the workflow right).